### ISSUE TITLE
Feature request: copy-paste friendly CLI

### ISSUE THREAD

**Issue by ConradIrwin** [kubernetes/kubernetes#31820]
**Is this a request for help?** no

**What keywords did you search in Kubernetes issues before filing this one?** "copy-paste"

---

The CLI is really slow to use right now, and one thing keeps tripping me up:

I run `kubectl get pods`, and see a list of pods. 
```
NAME                                             READY     STATUS    RESTARTS   AGE
auth-proxy-3lhje                                 1/1       Running   0          35d
backend-9737fb380f8dd8cb62f50988b66dae97-4inye   1/1       Running   0          3h
```

Then I copy-paste a pod name into `kubectl describe` and it complains at me: 
```
the server doesn't have a resource type "backend-9737fb380f8dd8cb62f50988b66dae97-4inye
```

One trivial change to the output format would fix this, which is to include the namespace whenever the CLI prints out a name:

```
$ kubectl get pods
NAME                                             READY     STATUS    RESTARTS   AGE
pod/auth-proxy-3lhje                                 1/1       Running   0          35d
pod/backend-9737fb380f8dd8cb62f50988b66dae97-4inye   1/1       Running   0          3h
```

This would then let me trivially copy-paste things around between commands without having to worry about missing namespaces.

Obviously the other way to fix this would be to observe that in most cases names are unique across all namespaces (and if there was an incentive to do this, that would become more true) and have commands do something sensible in the absense of a namespace.

```
$ kubectl describe backend-9737fb380f8dd8cb62f50988b66dae97-4inye
No resource type backend-9737fb380f8dd8cb62f50988b66dae97-4inye, showing pod/9737fb380f8dd8cb62f50988b66dae97-4inye instead.
...
```

**Comment by janetkuo** [comment]
`kubectl get pods --show-kind` gives you what you want.

**Comment by janetkuo** [comment]
This is already fixed by #28509. In the upcoming release 1.4, `kubectl get pods --show-kind` will give you what you want.
